tests/conftest.py

```diff
@@ -16,3 +23,50 @@ def app():
 @pytest.fixture
 def client(loop, app, sanic_client):
     return loop.run_until_complete(sanic_client(app))
+
+
+@pytest.fixture(scope="session")
+def engine():
+    # Test Database
+    TEST_DATABASE_NAME = getenv("TEST_DATABASE_NAME", "test_database")
+    TEST_DATABASE_HOST = getenv("TEST_DATABASE_HOST", "test_psql")
+    TEST_DATABASE_USER = getenv("TEST_DATABASE_USER", "test_user")
+    TEST_DATABASE_PASSWORD = getenv("TEST_DATABASE_PASSWORD", "test_password")
+    TEST_DATABASE_URL = f"postgresql://{TEST_DATABASE_USER}:{TEST_DATABASE_PASSWORD}@{TEST_DATABASE_HOST}/{TEST_DATABASE_NAME}"
+    return create_async_engine(TEST_DATABASE_URL)
+
+
+@pytest.fixture()
+async def tables(engine):
+    async with engine.begin() as conn:
+        await conn.run_sync(Base.metadata.create_all)
```

Comment:
I prefer to migrate or use schema + migrate to spot any issues regarding the migrations. I guess this will do for now. The tables are persisted though, since you do not call `drop_all` anywhere. Not a major issue, if we are using a new DB every time.

Reply:
Performing the migration sounds like a sensible thing to do.   I will see about changing this.